doc/docs/master/deploy-manage/deploy/deploy-pachyderm-jupyterhub.md

```diff
@@ -1,25 +1,34 @@
-# Deploy Pachyderm with JupyterHub
+# Deploy the Pachyderm IDE
 
 !!! note
-    JupyterHub integration with Pachyderm is an
-    enterprise feature. Contact sales@pachyderm.com
-    to request enabling JupyterHub integration
-    for your Pachyderm Enterprise license.
-
-JupyterHub is an open-source platform that enables you
-to spin multiple instances of single-user Jupyter notebook
+    The Pachyderm IDE is an enterprise feature,
+    which is also avaialble for testing during
+    the 14-day free trial.
+    Contact sales@pachyderm.com for more
+    information.
+
+The Pachyderm Integrated Development Environment (IDE) is
+an extension to the standard Pachyderm deployment that
+enables you to leverage JupyterHub and JupyterLab user interface
+integrated with the Pachyderm data version control system.
```

Comment:
I feel like the following two paragraphs go into too much detail of JH/JL.

What about nixing them, and replacing this paragraph with something like this:

```
The Pachyderm Integrated Development Environment (IDE) is
an optional extension to Pachyderm clusters that provides a
comprehensive environment for prototyping and deploying
Pachyderm pipelines, as well as introspecting on data in the
Pachyderm version control system. It builds off of Jupyter,
JupyterHub, and JupyterLab - familiar tools for many data
scientists.
```

Reply:
I like this! Rephrased a bit and added.